src/studio/nuxeo-studio/working-in-studio/migration-services.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Migration Services
+review:
+    comment: ''
+    date: '2018-12-05'
+    status: ok
+tree_item_index: 950
+toc: true
+---
+The JSF UI to Web UI migration tool is a safe and automated way to create, within a few minutes, an equivalent Web UI configuration based on your existing JSF UI configuration.
+
+The migration tool reduces dramatically the effort needed to transition to [Web UI]({{page space="nxdoc" page="web-ui"}}), our user interface that uses modern web standard technologies to provide an outstanding user experience.
+
```

Comment:
Before this section, we need to indicate where to find the Migration service (adding a screenshot also)

Reply:
As the page is now under the Studio Modeler section, and as we don't indicate it in the other pages, could we put a screenshot underlined ? (see commit "implement feedback" to come)